pkg/scaffold/controller/controller.go

```diff
@@ -205,6 +205,10 @@ type Reconcile{{ .Resource.Kind }} struct {
 func (r *Reconcile{{ .Resource.Kind }}) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	// Fetch the {{ .Resource.Kind }} instance
 	instance := &{{ .Resource.Group}}{{ .Resource.Version }}.{{ .Resource.Kind }}{}
+{{- if not .Resource.Namespaced }}
+	// For cluster-scoped CRDs, we force the namespace to be empty.
+	request.NamespacedName.Namespace = ""
```

Comment:
This seems like a controller-runtime bug if this isn't empty already

Reply:
we know in CR what the scope is -- we should be able to not have to do this -- can you file a bug against CR?

Reply:
It's a `controller-runtime` issue but the patch cannot be introduced due to dependency hell :(